app/assets/stylesheets/helpers/_inverse-background.scss

```diff
@@ -0,0 +1,4 @@
+.inverse-background {
+    background: $govuk-brand-colour;
+    color: govuk-colour("white");
```

Comment:
Reminder - I don't think you need to specify the color (just the background) here.